projects/storefrontstyles/scss/components/myaccount/_my-interests.scss

```diff
@@ -0,0 +1,222 @@
+%cx-my-interests {
+  .cx-product-interests-title {
+    margin: 0 auto;
+  }
+  .cx-product-interests {
+    @include media-breakpoint-down(sm) {
+      max-width: 1%;
+      padding: 0;
+    }
+  }
+  .cx-product-interests-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-width: 0 0 1px 0;
+    border-color: var(--cx-color-light);
+    border-style: solid;
+
+    tr {
+      width: 100%;
+
+      @include media-breakpoint-down(sm) {
+        border-width: 1px 0 0 0;
+        border-color: var(--cx-color-light);
+        border-style: solid;
+
+        &:first-child {
+          border-width: 1px 0 0 0;
+          padding: 1.25rem 0 0 0;
+        }
+      }
+    }
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      text-align: left;
+
+      &:first-child {
+        text-align: center;
+      }
+    }
+
+    td {
+      width: 25%;
+      padding: 1.625rem 0;
+
+      &:first-child {
+        width: 10%;
+        padding-right: 1.25rem;
+        padding-left: 1.25rem;
+      }
+      &:last-child {
+        justify-content: flex-end;
+      }
+
+      @include media-breakpoint-up(md) {
+        &:last-child {
+          text-align: center;
+        }
+      }
+
+      .cx-name {
+        a {
+          color: $dark;
```

Comment:
The use of SASS variables for colors has been discontinued. All color values should be assigned as **CSS variables** therefore: `$dark` should be `--cx-color-dark`

![Screen Shot 2019-11-21 at 11 56 11 AM](https://user-images.githubusercontent.com/15113219/70110572-6b670e00-161d-11ea-83fd-e12a286be55f.png)

Reply:
Updated.